my-app/src/App.css

```diff
@@ -0,0 +1,112 @@
+* {
+  font-family: "Lato", sans-serif;
+  margin: 0;
+  padding: 0;
+  /* font-size: 12px; */
+}
+
+body,
+html {
+  margin: 0;
+  padding: 0;
+}
+
+.post {
+  background: #ffffff;
+  font-family: "Lato", sans-serif;
```

Comment:
This was already defined in the catch-all rule (same for `.btn`). Maybe I'm missing something, but I can't detect any parent element having a different font-family - it seems fine to remove it.

Reply:
Yep the whole file will inherit the font-family from *. Removing redundant font-family styles.